src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -99,24 +101,25 @@ public ArmSubsystem() {
   public void setPosition(double angleDegrees) {
     targetDegrees = angleDegrees;
     armMotor.setControl(
-        new MotionMagicVoltage(Constants.Arm.ANGLE_TO_ENCODER_ROTATIONS(angleDegrees)));
+        controlRequest.withPosition(Constants.Arm.ANGLE_TO_ENCODER_ROTATIONS(angleDegrees)));
   }
 
-  private double getDegrees() {
+  private double calculateDegrees() {
     return revEncoder.get() * 360d;
   }
 
   public boolean atTarget(double endToleranceDegrees) {
-    if (getDegrees() < targetDegrees + endToleranceDegrees
-        && getDegrees() > targetDegrees - endToleranceDegrees) {
+    if (encoderDegrees < targetDegrees + endToleranceDegrees
+        && encoderDegrees > targetDegrees - endToleranceDegrees) {
```

Comment:
Here, instead of reading the value from the encoder twice every time `atTarget()` is called, we use the stored value.